recipes/planetmapper/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "planetmapper" %}
+{% set version = "1.7.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/planetmapper-{{ version }}.tar.gz
+  sha256: fa1647de939d555824556f6d2de7ed0f913912ed6a3a178030dfe06518fdf390
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - planetmapper = planetmapper:main
+
+requirements:
+  host:
+    - python >=3.10
+    - pip
+  run:
+    - python >=3.10
+    - astropy
+    - matplotlib-base
+    - numpy
+    - pillow
+    - spiceypy
+    - scipy
+    - photutils
+    - tqdm
+    - pyproj
+    - typing-extensions
+    - xorg-libx11
+
+test:
+  imports:
+    - planetmapper
+  commands:
+    - pip check
```

Comment:
You could add a test command `planetmapper --help` if that exists, or `planetmapper --version`

Reply:
planetmapper doesn't currently have any commands that don't launch a GUI, but I'll remember to add a test command if/when we add e.g. `planetmapper --help` functionality